apps/explorer/vite.config.ts

```diff
@@ -1,14 +1,45 @@
 /// <reference types="vitest" />
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+import { defineConfig, searchForWorkspaceRoot } from 'vite'
+import react from '@vitejs/plugin-react-swc'
 import viteTsConfigPaths from 'vite-tsconfig-paths'
+import { ModuleNameWithoutNodePrefix, nodePolyfills } from 'vite-plugin-node-polyfills'
+import stdLibBrowser from 'node-stdlib-browser'
+import { loadConfig } from './loadConfig'
+
+import { version as APP_VERSION } from './package.json'
+import { version as CONTRACT_VERSION } from '@cowprotocol/contracts/package.json'
+import { version as DEX_JS_VERSION } from '@gnosis.pm/dex-js/package.json'
+
+const CONFIG = loadConfig()
+
+const allNodeDeps = Object.keys(stdLibBrowser).map((key) => key.replace('node:', '')) as ModuleNameWithoutNodePrefix[]
+
+// Trezor getAccountsAsync() requires crypto and stream (the module is lazy-loaded)
```

Comment:
Why do we need trezor setup here?

Reply:
Copy-paste from Cowswap, thanks! Removed it